.github/workflows/test-suite.yml

```diff
@@ -14,7 +14,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11-dev"]
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11.0-rc.1"]
```

Comment:
Not really sure if `3.11-dev` will include `rc` releases or not. And the `3.11.0-rc.1` is not yet available in GH actions.

Reply:
`-dev` includes all releases, not only alpha and beta.

Reply:
ref.: https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#python

Reply:
It's kind of ambiguous, it does mention alpha and beta but not RC. But I guess you're right.